diff --git a/CMakeLists.txt b/CMakeLists.txt
index f0f9074..de582db 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -37,11 +37,11 @@ if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
     if ("${CMAKE_CXX_COMPILER_VERSION}" VERSION_LESS "4.8")
         message(SEND_ERROR "Version ${CMAKE_CXX_COMPILER_VERSION} of the ${CMAKE_CXX_COMPILER_ID} C++ compiler is not supported. Please use version 4.8 or later.")
     endif()
-
-    # Adjusting CMAKE_C_FLAGS to get wayland protocols to compile
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu90")
 endif()
 
+# Adjusting CMAKE_C_FLAGS to get Wayland protocols to compile
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu90")
+
 # Add custom uninstall target
 if ("${ECM_VERSION}" VERSION_LESS "1.7.0")
     # ECM 1.7.0 provides this target, so we can't roll our own
